metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, config: FivetranSourceConfig, ctx: PipelineContext):
 
         # Create and register the stateful ingestion use-case handler.
         self.stale_entity_removal_handler = StaleEntityRemovalHandler.create(
-            self, self.config, self.ctx
+            self, self.config, ctx
```

Comment:
just for my understanding - why was this needed?

Reply:
My Python extension was not showing that self have ctx variable. Hence I changed it to use directly ctx.
But now I debug and got to know that source class have ctx variable which we are initializing at fivetran.py:69.
Hence I am reverting this changes.